docs/monitoring/README.rst

```diff
@@ -0,0 +1,29 @@
+:orphan:
+
+This README explains the purpose of this directory in the form of a decision document.
+
+Context
+-------
+
+At this time, we don't have a standard way to pull in rst docs from outside of the main docs directory. See https://github.com/sphinx-doc/sphinx/issues/701 for details on this issue.
+
+Decision
+--------
+
+Copy docs files and use an include statement as a quick temporary solution with the following benefits:
+
+* Any old github references to the old locations would still be accurate.
+* We may be able to implement a more automated solution in the future that doesn't break the new Readthedoc locations.
+
+Consequences
+------------
+
+New docs need to be copied to be included in the index.rst, which is only one minor additional step.
+
+Rejected Alternatives
+---------------------
+
+The following alternatives were temporarily rejected for the sake of expediency. The decision could be updated in more time were invested on this issue more globally.
+
+#. Move the monitoring/docs folder under the main docs folder. This would break existing references to github docs. Someone could choose to make this change in the future.
```

Comment:
What about having the includes go the other direction, with e.g. `edx_django_utils/monitoring/docs/decisions/0001-monitoring-by-code-owner.rst` including `../../../../docs/monitoring/decisions/0001-monitoring-by-code-owner.rst`? We generally have decisions under the `docs` root dir anyhow.

Reply:
This was addressed as discussed, by mentioning we wanted to keep the docs near the code. See https://github.com/edx/edx-django-utils/pull/128#discussion_r635297807 for details.